tests/kickstart_tests/escrow-cert.ks

```diff
@@ -0,0 +1,107 @@
+%pre
+# Create an nss database for the escrow certifcate
+mkdir -p /tmp/escrow_test/nss
+certutil -d /tmp/escrow_test/nss --empty-password -N
```

Comment:
Does the database need to be created for the escrow to work or is it only needed for the test to verify the results? IOW, do we require people using escrow having this in their kickstarts?